ra-optimisation/rao-commons/src/main/java/com/farao_community/farao/rao_commons/RaoUtil.java

```diff
@@ -137,14 +137,15 @@ public static IteratingLinearOptimizer createLinearOptimizer(RaoParameters raoPa
             fillers.add(new MaxMinRelativeMarginFiller(raoParameters.getObjectiveFunction().getUnit(), raoParameters.getPstPenaltyCost(), raoParameters.getNegativeMarginObjectiveCoefficient(), raoParameters.getPtdfSumLowerBound()));
             fillers.add(new MnecFiller(raoParameters.getObjectiveFunction().getUnit(), raoParameters.getMnecAcceptableMarginDiminution(), raoParameters.getMnecViolationCost(), raoParameters.getMnecConstraintAdjustmentCoefficient()));
         }
+        fillers.add(new OperatorsNotToOptimizeFiller(operatorsNotToOptimize));
```

Comment:
Maybe just do it if the parameter (don't remember its name :p) is `true`

It's a bit redundant with the fact that the operatorsNotToOptimize will be empty in that case, but it would avoid loosing time in this filler.

Reply:
I agree, but the parameter is specific to the curative RAO, and it's in the SearchTreeParameters extension.
The corresponding parameter in the TreeParameters is this Set, which can be null for preventive or for curative if the parameter is false.
To save time, we can skip this if the set is empty or null ?

Reply:
>  To save time, we can skip this if the set is empty or null ?

Yeah, why not

Reply:
done